benches/cub/device/select/if.cu

```diff
@@ -0,0 +1,176 @@
+#include <nvbench/nvbench.cuh>
```

Comment:
Let's split this and `if_complex` into

```
benches/cub/device/select/if/basic.cu
benches/cub/device/select/if/complex_predicate.cu
```

to keep the generated test/executable names consistent.